scripts/pin_the_slow_way.py

```diff
@@ -29,22 +29,22 @@
 
 
 pinned = {
-          'boost': 'boost 1.60.*',
-          'bzip2': 'bzip2 1.0.*',
-          'fontconfig': 'fontconfig 2.11.*',
+          'boost': 'boost 1.61*',
```

Comment:
It won't. We have 1.61.0 so that will get installed instead. We pinned everything that needed pinning to 1.60.* so that they would not be broken before we released 1.61.0. Now its time to try an upgrade to see if they are fixed.